src/lib/axl-explorations/src/components/games/LetterLauncherGameStoryPreview.tsx

```
@@ -244,11 +244,11 @@ export function LetterLauncherGameStoryPreview({
 
   // Generate practice questions
   const generatePracticeQuestion = (): LetterLauncherQuestion => {
-    const supportedLanguage: 'en' | 'te' | 'mr' | 'kn' | 'hi' = 
-      (contentLanguage === 'en' || contentLanguage === 'te' || contentLanguage === 'mr' || contentLanguage === 'kn' || contentLanguage === 'hi' ) 
-        ? contentLanguage 
+    const supportedLanguage: 'en' | 'te' | 'mr' | 'kn' | 'hi' =
```

ESLint

[error] Replace `'en'·|·'te'·|·'mr'·|·'kn'·|·'hi'` with `"en"·|·"te"·|·"mr"·|·"kn"·|·"hi"`

(prettier/prettier)